examples/stable/mysql-csi-snapshot/README.md

```diff
@@ -0,0 +1,267 @@
+actionset backup-mlvcv created
+
+$ kubectl --namespace kanister get actionset
+NAME                         AGE
+backup-mlvcv                 112s
+
+# View the status of the actionset
+# Please make sure the name of the actionset here matches with name of the actionset created above
+$ kubectl --namespace kanister describe actionset backup-mlvcv
+
+# Check the CSI VolumeSnapshot created
+$ kubectl -n mysql get volumesnapshot
+NAME                                  READYTOUSE   SOURCEPVC              SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS      SNAPSHOTCONTENT                                    CREATIONTIME   AGE
+data-mysql-release-0-snapshot-cd26z   true         data-mysql-release-0                           8Gi           do-block-storage   snapcontent-2e411d1d-0b5f-48d4-9b79-b19e824c2e38   3h40m          3h40m
+```
+
+## Disaster strikes!
+
+Let's say someone accidentally deleted the test database.
+
+```bash
+# Connect to MySQL by running a shell inside MySQL's pod
+$ kubectl exec -it $(kubectl get pods -n mysql --selector=app.kubernetes.io/instance=mysql-release -o=jsonpath='{.items[0].metadata.name}') -n mysql -- bash
+
+$ mysql --user=root --password=asd#45@mysqlEXAMPLE
+
+mysql> SHOW DATABASES;
++--------------------+
+| Database           |
++--------------------+
+| information_schema |
+| my_database        |
+| mysql              |
+| performance_schema |
+| sys                |
+| test               |
++--------------------+
+6 rows in set (0.00 sec)
+
+mysql> DROP DATABASE test;
+Query OK, 1 row affected (0.12 sec)
+
+mysql> SHOW DATABASES;
++--------------------+
+| Database           |
++--------------------+
+| information_schema |
+| my_database        |
+| mysql              |
+| performance_schema |
+| sys                |
++--------------------+
+5 rows in set (0.01 sec)
+```
+
+## Restore MySQL DB
+
+To restore the missing data, you should use the backup that you created before. We will use `kanctl` for creating this restore action.
+
+```bash
+# Make sure to use correct backup actionset name here
+$ kanctl --namespace kanister create actionset --action restore --from backup-mlvcv
+actionset restore-backup-mlvcv-6z9xn created
+
+# Check the PVCs in mysql namespace. You should see a new PVC in the format '<PVC-name>-restored'
+$ kubectl -n mysql  get pvc
+NAME                            STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+data-mysql-release-0            Bound    pvc-b966c5ef-3d20-4ec2-9ab3-1a868737ea0a   8Gi        RWO            do-block-storage   4m14s
+data-mysql-release-0-restored   Bound    pvc-2eb6fb89-78bc-4c2e-8c1e-c270d664f311   8Gi        RWO            do-block-storage   18s
+
+# Use following command to debug if a new PVC in the format '<PVC-name>-restored' is not created
+$ kubectl --namespace kanister describe actionset restore-backup-mlvcv-6z9xn
+```
+
```

Comment:
Not very confident with this approach. Here's what I think
1. Somewhere above, we should probably add a note, saying that this process may not work if application-consistent backups are required.
2. Recommend scaling down the workload (SS) deleting the existing PVC and restoring one with the same name above, and finally scaling up the workload to achieve a consistent restore.